changelog.d/5969.misc

```diff
@@ -0,0 +1 @@
+Change account_threepid_delegate to an account_threepid_delegates dictionary to handle the case of Synapse handling email verification, and an external server handling msisdn verification.
```

Comment:
given that `account_threepid_delegate` has never been released, this doesn't really make sense.

It probably makes sense to change the 5876.misc and make this match it?

(also, these aren't really `misc` changes, since they affect the synapse admin. `feature` is a better match).